app/views/merchant_items/edit.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Edit Item</h1>
+<%= form_with url: "/merchants/#{@merchant.id}/items/#{@item.id}", method: :patch, local: true do |f| %>
```

Comment:
I'm not sure if this is a thing, but is this a spot where we could use advanced routing too?